components/procore/sources/budget-snapshot/budget-snapshot.js

```diff
@@ -0,0 +1,50 @@
+const common = require("../common.js");
+
+module.exports = {
+  ...common,
+  name: "New or Updated Budget Snapshot (Instant)",
```

Comment:
The name should be just "Budget Snapshot Event (Instant)" instead, since it includes new, updated or deleted snapshots, right?

(This is repeated in multiple places so I'm making the comment here, but it applies to other components)